README.md

```diff
@@ -1,7 +1,8 @@
-# Ice Window Manager (IceWM)
+# Ice Window Manager (IceWM) Research for fix bug : uncorrectly size screen when we using multimonitor and xrandr
```

Comment:
More detail about this bug, please do as below step:
1. Plug second screen (by VGA or HDMI)
2. Using xrandr to extend screen:
    xrandr --output VGA --right-of HDMI --output HDMI --auto
3. Using this command to switch between them
   xrandr --output VGA --preferred --left-of HDMI --output HDMI --preferred
in this time 2 screen will be wrap but not resize correctly.
The login of current icewm code in only check big size (width(), height() ) but not care about each screen => it is bad in this stuation.